serve.json

```diff
@@ -11,7 +11,7 @@
       ]
     },
     {
-      "source": "{**/*.js,**/*.css}",
+      "source": "**/*-@(????????????????????????????????|????????????????????).@(js|woff|ttf|svg|png)",
```

Comment:
This is horrible and I love it.

Globs don't support a "match N times" like regex, so I don't know of a better approach here unfortunately.

Reply:
I kept looking for the magic syntax to pull that off, but, yeah,  not supported.

Also, I think you'd have liked one of my attempts. 32 copies of `[a-z0-9]` followed by `|` followed by 20 more.

Serve said no 😞